ci_handbook/data_format.qmd

```diff
@@ -0,0 +1,12 @@
+---
+title: "{{< fa arrow-right >}} Data Format"
```

Comment:
'Data Standards' may  be a more recognizable heading for this section

Reply:
updated